phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/container/control/ContainersFeaturePanel.java

```diff
@@ -54,9 +55,15 @@
 
     /**
      * The container engine controller
+     * TODO: remove and directly use `enginesManager` instead
```

Comment:
Is this something you want to fix before merging?

Reply:
Done